app/application/template.hbs

```diff
@@ -1,38 +1,33 @@
-    <section id="feed-registry" class="jumbotron">
-      <div class="container">
-
-      {{outlet}}
-
-      </div>
-
-    </section>
-    <footer>
-      <div class="container">
-        <div class="col-sm-4 col-sm-offset-1 col-xs-5 col-xs-offset-1">
-            <ul>
-              <h6> technology </h6>
-              <li><a href="/how-it-works">How It Works</a></li>
-              <li><a href="/feed-registry">Feed Registry</a></li>
-              <li><a href="https://github.com/transitland/transitland-datastore/blob/master/README.md#api-endpoints">Datastore API</a></li>
-              <li><a href="/playground">Playground</a> </li>
-            </ul>
-          </div>
-          <div class="col-sm-4 col-xs-6">
-            <ul >
-              <h6> resources </h6>
-              <li><a href="/an-open-project">Data Licensing</a></li>
-              <li><a href="/news">News & Updates </a></li>
-            </ul>
-          </div>
-          <div class="col-sm-3 col-xs-6">
-            <ul>
-              <h6> participate </h6>
-              <li>{{fa-icon icon="smile-o"}} <a href="/participate">How to Participate</a></li>
-              <li> {{fa-icon icon="github-alt"}} <a href="//github.com/transitland">GitHub</a></li>
-              <li> {{fa-icon icon="twitter"}} <a href="//twitter.com/transitland">Twitter</a> </li>
-              <li> {{fa-icon icon="envelope-o"}} <a href="mailto:transitland@mapzen.com">Email</a></li>
-            </ul>
-          </div>
-          <p class="center-text"><em>Transitland is sponsored by <a href="https://mapzen.com">Mapzen</a>.</em></p>
-        </div>
-    </footer>
\ No newline at end of file
+		<section id="feed-registry" class="jumbotron">
+			{{outlet}}
```

Comment:
'add a feed' section doesn't need to be nested in the container because of suv nav, so I took container class out from template page put them in the table view